Assets/Scripts/EnemySpawnManager.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemySpawnManager : MonoBehaviour
+{
+    // Game object that the spawner will be attached to
+    [SerializeField] GameObject attachedGameObject;
+    [SerializeField] float distanceToAttachedObject;
+    [SerializeField] private GameObject enemyPrefab;
+    [SerializeField] private float startDelay;
+    [SerializeField] private float repeatedRate;
+    [SerializeField] private float minYPos;
+    [SerializeField] private float maxYPos;
+
+    private void Start()
+    {
+        InvokeRepeating("SpawnEnemy", startDelay, repeatedRate);
+    }
+
+    void Update()
+    {
+        AdjustSpawnerPosition();
+    }
+
+    void SpawnEnemy()
+    {
+        Transform gameObjectPos = gameObject.transform;
+        float randomYPos = Random.Range(minYPos, maxYPos);
+        Instantiate(enemyPrefab, new Vector3(gameObjectPos.position.x, randomYPos, gameObjectPos.position.z), Quaternion.identity);
```

Comment:
Let's change this to `enemyPrefab.transform.position.x, randomYPos, enemyPrefab.transform.position.z), Quaternion.identity, gameObject.transform`

The `gameObject.transform` parameter sets the parent transform to the game object this script is attached to (which is the game object Spawner). This allows enemies to spawn nested within the parent to keep a little organized.

We can then change the enemy prefab's x transform to 10.0 so that all enemies spawn there.